rootfs/Dockerfile

```diff
@@ -1,17 +1,14 @@
-FROM quay.io/deis/base:v0.3.6
+FROM nsqio/nsq:v1.1.0
 
-RUN adduser --system \
-      --shell /bin/bash \
-      --disabled-password \
-      --home /opt/nsq \
-      --group \
-      nsq \
-    && curl -Ls https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 > /usr/bin/jq \
-    && chmod +x /usr/bin/jq \
-    && curl -sL https://github.com/nsqio/nsq/releases/download/v1.1.0/nsq-1.1.0.linux-amd64.go1.10.3.tar.gz | tar xz \
-    && mv /nsq-1.1.0.linux-amd64.go1.10.3/bin/* /bin \
-    && rm -r /nsq-1.1.0.linux-amd64.go1.10.3 \
-    && mkdir /opt/nsq/data
+RUN mkdir -p /opt/nsq/data \
+    && adduser \
```

Comment:
Can we use the more verbose flags `--shell, --disabled-password, ...`? I think it makes sense here because it makes things more clear and if a user does not know the flags of adduser they do not have to look them up.

Reply:
I'm sorry about this, because I also like to have clear parameters. But the shell contained by busybox is very limited, and it does not support it.

Create new user, or add USER to GROUP

-h DIR		Home directory
-g GECOS	GECOS field
-s SHELL	Login shell
-G GRP		Group
-S		Create a system user
-D		Don't assign a password
-H		Don't create home directory
-u UID		User id
-k SKEL		Skeleton directory (/etc/skel)

Reply:
We will have to live with this limitation for now. A good tradeoff for not having to maintain the base image. :+1: